apps/docs/generator/src/main.ts

```diff
@@ -84,4 +85,32 @@ function generateValueTypeDocs(rootPath: string): void {
   console.info(`Generated file ${fileName}`);
 }
 
+function generateExampleDocs(rootPath: string): void {
+  const docsPath = join(rootPath, 'apps', 'docs', 'docs', 'user', 'examples');
+  const examplesPath = join(rootPath, 'example');
+
+  for (const file of readdirSync(examplesPath)) {
+    if (file.endsWith('.jv')) {
+      const exampleFilePath = join(examplesPath, file);
+      const exampleModel = readFileSync(exampleFilePath);
+
+      const exampleName = file.slice(0, -3);
```

Comment:
Controversial opinion: Write `-'.jv'.length()` instead of `-3` to avoid a magic number.

Reply:
Very expressive, I like it!